src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    public static final String NULL_VALUE = null;
+    public static final String VALID_NAME = "Tom123";
+    public static final String VALID_PASSWORD = "aBc321";
+    public static final Integer VALID_AGE = 22;
+
+    private static RegistrationService registrationService;
+    private static User actualUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        actualUser = new User(VALID_NAME, VALID_PASSWORD, VALID_AGE);
+    }
+
```

Comment:
Also, you can add the @AfterAll method, which will clear the storage - Storage.people.clear();

Reply:
I thought about it. Maybe it is better to add aftereach method that will allow to run every test method with clean storage?

Reply:
it's valid too